confidant/scripts/archive.py

```diff
@@ -0,0 +1,93 @@
+import sys
+import logging
+from datetime import datetime
+
+from flask_script import Command, Option
+
+from confidant import settings
+from confidant.models.credential import Credential, CredentialArchive
+from confidant.services import servicemanager
+
+logger = logging.getLogger(__name__)
+
+logger.addHandler(logging.StreamHandler(sys.stdout))
+logger.setLevel(logging.INFO)
+
+
+class ArchiveCredentials(Command):
+
+    option_list = [
+        Option('--days', dest='days', required=True, type=int),
+        Option('--force', action='store_true', dest='force', default=False),
+    ]
+
+    def run(self, days, force):
```

Comment:
you might want to consider adding a sleep or something to make sure you don't blow past your WCU

Reply:
This is a good point. I think I'm going to make a function for deleting and saving that will catch exceptions for provisioned write and will backoff and retry.